recipes/r-geogrid/meta.yaml

```
@@ -0,0 +1,103 @@
+{% set version = '0.1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-geogrid
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/geogrid_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/geogrid/geogrid_{{ version }}.tar.gz
+  sha256: 07c54dac348200b5dc4c610fce643d672f1e273ebe8802bff9a6f4241f770a2d
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{posix}}filesystem        # [win]
```

```suggestion
    - {{ posix }}filesystem        # [win]
```
Same for the other lines.